src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
+  label: string;
+  path: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface ComponentInstance {
+  component: ExtendedComponentSchema;
+  getView: any;
+  id: string;
+  type: string;
+  asString?(value: any): string;
+}
+
+export interface GridHeader {
+  component: ComponentInstance | null;
+  key: string;
+  label: string;
+  renderCell?(cellValue: any, component?: any): string;
+}
+
+export interface FormRow {
+  form: string;
+  project: string;
+  created: string;
+  modified: string;
+  owner: string;
+  data: object;
```

Comment:
I'd need to discuss with you what `FormRow` is used for in order to make sure properties' types are correct